core/src/main/java/hudson/triggers/SlowTriggerAdminMonitor.java

```diff
@@ -28,6 +29,7 @@
     @NonNull
     private final Map<String, Value> errors = new ConcurrentHashMap<>();
 
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
     public static int MAX_ENTRIES = 10;
```

Comment:
Maybe makes sense to add a system property support. Not a blocker

Reply:
Done. I guess adding this to the documentation would make sense ...